README.md

```diff
@@ -64,3 +64,96 @@ It may be possible to lower the "admin" privilage levels to a "creator" level if
 ## Development
 
 Please submit any feature enhancements, bug fixes, or ideas via pull requests or issues.
+
+# Terraform Docs
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.12 |
```

Comment:
this is wrong, we are using 1.x.y

Reply:
I took this from the `test` requirements. will update